kitsune/products/models.py

```diff
@@ -134,6 +135,9 @@ class Topic(ModelBase):
 
     # Whether or not this topic is visible in the ui to users.
     visible = models.BooleanField(default=False)
+    # Whether or not this topic is used in the AAQ.
+    in_aaq = models.BooleanField(
+        default=False, help_text=_lazy(u'Whether this topic is shown to users in the AAQ or not.'))
```

Comment:
This is only used in the new AAQ, not the old one, so it isn't going to have any effect on the actively used part of production. I wanted it to be consistent with `visible`.